user-manual/transfer/forensic.rst

```diff
@@ -4,130 +4,169 @@
-If you are processing disk images, you will want to adjust the Archivematica
-default configuration options in the Administration tab of the dashboard for
-some decision points in the workflow. Archivematica default is to extract
-content from any packages, which includes forensic image formats as well as
-compressed content like zip files, and to then delete the package itself (though
-the metadata and logs for the package are retained in the metadata and logs). To
-make these decisions in the dashboard processing workflow:
+* :ref:`Workflow configuration options <disk-image-configuration-options>`
 
-* Deselect 'Extract packages' - If you find you are selecting YES or NO most
-  of the time, you can preconfigure this choice accordingly to YES or NO in
-  the administration tab of the dashboard instead.
+* :ref:`Using the disk image transfer type <disk-image-workflow>`
 
-* Deselect 'Examine contents' - If you find you are selecting YES or NO most
-  of the time, you can preconfigure this choice accordingly to YES or NO in
-  the administration tab of the dashboard instead.
+  * :ref:`Compound disk images <compound-disk-images>`
 
-* Deselect 'Delete packages after extraction' - If you find you are selecting
-  YES or NO most of the time, you can preconfigure this choice accordingly to
-  YES or NO in the administration tab of the dashboard instead.
+.. _disk-image-configuration-options:
 
-.. seealso::
+Workflow configuration options
+------------------------------
 
-   :ref:`Processing configuration <dashboard-processing>`
+If you are processing disk images, you may want to adjust Archivematica's
+:ref:`processing configuration <dashboard-processing>` options in the
+Administration tab of the dashboard.
 
+For more information on the processing configuration fields, see
+:ref:`Fields and options <processing-config-fields>`.
 
-Upload image to dashboard
--------------------------
+Extract packages
+++++++++++++++++
 
-1. Select Disk Image type from the drown-down menu on the Transfer tab of the
-   dashboard.
+Archivematica can be configured to extract the contents from a package, which
+includes forensic image formats as well as compressed content like ZIP files.
 
-2. Add metadata from the imaging process if desired by clicking on the report
-   button.
+However, extracting the contents of a forensic disk image can cause major
+performance issues, since disk images often contain many thousands of system
+files that need to be subsequently processed in Archivematica. This can result
+in failures. We recommend not extracting forensic image formats. This can be set
+in the processing configuration by setting the *Extract packages* job to "No".
 
-.. figure:: images/forensic-start.*
-   :align: center
-   :figwidth: 70%
-   :width: 100%
-   :alt: Click on the report button to add metadata
+If you would like to extract forensic image formats anyway, you can set the
+*Extract packages* job to "Yes". However, we recommend testing this at scale to
```

Comment:
I think we can eliminate this second "However" in this section and just start with "We recommend testing..."